P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+        (Twitter) timelines, comments on Facebook pages, posts in forum threads, and comments or
+        contributions to discussions on Wikipedia talk pages or in the comment sections of
+        weblogs.</p>
+      <p>Posts can be either written or spoken: <list>
+          <item><emph rend="bold">written</emph> or <emph rend="bold">multimodal posts</emph>: In
+            the majority of CMC technologies posts are composed as stretches of text using a
+            keyboard or speech-to-text conversion software in an entry form on the screen. In
+            many cases the technology allows authors to include or embed graphics (emojis or
+            images), video files, and hyperlinks into their posts.</item>
+          <item><emph rend="bold">audio posts</emph>: A growing number of CMC technologies, e.g.
+            messenger software such as WhatsApp or RocketChat, allow for an alternative, spoken
+            production of posts by providing a recording function which allows users to record a
+            stretch of spoken language and transmit the resulting audio file to the other
+            parties.</item>
+        </list></p>
+      <p>The element <gi>post</gi> may co-occur with <gi>u</gi>, <gi>kinesic</gi>,
+        <gi>incident</gi>, or other existing TEI elements within a <gi>div</gi>, or directly within
+        the <gi>body</gi>, and may contain headings, paragraphs, openers, closers, or
+        salutations.</p>
+      <p>The <gi>post</gi> element is a member of the TEI attribute classes <ident type="class"
+          >att.ascribed</ident>, <ident type="class">att.canonical</ident>, <ident type="class"
+          >att.datable</ident>, <ident type="class">att.global</ident>, <ident type="class"
+          >att.timed</ident>, and <ident type="class">att.typed</ident>, and as such may take a
+        variety of attributes. Common attributes used in conjunction with <gi>post</gi> include
+          <att>who</att>, <att>synch</att>, <att>type</att>, <att>subtype</att>, <att>rend</att>,
+        and <att>xml:id</att>.</p>
+    </div>
+    <div xml:id="CMCcmcpostatts">
+      <head>Attributes specific to CMC <gi>post</gi></head>
+      <p>Three attributes pertain specifically to <gi>post</gi>:
+        <specList>
+          <specDesc key="post" atts="modality replyTo indentLevel"/>
+        </specList>
+        The type of the content of a post (i.e., whether the content is written, an
+        image, a video clip, etc.) is indicated by the child elements of the <gi>post</gi>. (E.g., a
+        <gi>post</gi> might have a child <gi>p</gi>, or a child <gi>figure</gi> with a
+        <gi>graphic</gi>, or a child <gi>figure</gi> with a <gi>media</gi>, or some combination
+        thereof.) How that content was created — in particular whether it was recorded speech or not
+        — may be described with the <att>modality</att> attribute. Because spoken language differs
```

Comment:
Style: never start a sentence with "because". ;-)

Suggestion: As spoken language differs significantly from written language, ...
(mark the comma!).

Reply:
As stated above, “Because” is OK, but because it introduces a dependent clause a comma after the 2nd occurrence of “language” is required. I have added it in 759c2adba.